AirCasting/Utils/DataFormatters.swift

```diff
@@ -2,9 +2,47 @@
 //
 
 import Foundation
+import Resolver
 
 enum DateFormatters {
     
+    struct SessionCardView {
+        @InjectedObject private var userSettings: UserSettings
+        static let shared = SessionCardView()
+        
+        var pollutionChartDateFormatter: DateFormatter {
+            let df = DateFormatter()
```

Comment:
Applies to all methods in `SessionCardView` and `TimeAxisRenderer`

Creating date formatters is a heavy task, we should keep them static and only choose which one to use based on `userSettings`, not create them each time from scratch